.github/workflows/scripts/test-integrations.sh

```
@@ -196,61 +232,158 @@ fi
+  cd "$PYTHON_TEST_DIR"
+
+  if command -v uv >/dev/null 2>&1; then
+    echo "📦 Installing Python dependencies with uv..."
+    uv sync --frozen --quiet
+    PYTHON_TEST_COMMAND=(uv run pytest)
+  else
+    echo "⚠️  uv not found, trying pip..."
+    if [ ! -d ".venv" ]; then
+      python3 -m venv .venv
+    fi
+    source .venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: .venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)